README.adoc

```diff
@@ -118,33 +120,35 @@ cd query
+
+[role="no_copy"]
+----
+**************************************************************
+*    Liberty is running in dev mode.
+----
+
+Dev mode holds your command-line session to listen for file changes. Open another command-line session to continue, or open the project in your editor.
+
 // file 0
-system/server.xml
+query/pom.xml
 [source, XML, linenums, role='code_column']
 ----
-include::finish/system/src/main/liberty/config/server.xml[]
+include::finish/query/pom.xml[]
 ----
 
 // file 1
-query/SystemResource.java
+query/server.xml
 [source, XML, linenums, role='code_column']
 ----
-include::start/query/src/main/java/io/openliberty/guides/query/SystemResource.java[]
+include::finish/query/src/main/liberty/config/server.xml[]
 ----
 
-A [hotspot=basicRegistry file=0]`basicRegistry` element is configured in the `system` server configuration file that is located at `system/src/main/liberty/config/server.xml`. Only the registered user is authorized to access the `system` microservice. Note that the `basicRegistry` element is a simple case for learning purposes. For more information about the different user registries, see the https://openliberty.io/docs/latest/user-registries-application-security.html[User registries documentation^].
-
-The `/query/systems/{hostname}` endpoint constructs the [hotspot=authHeader file=1]`authHeader` authorization header with the [hotspot=user file=1]`system.user` and [hotspot=password file=1]`system.password` properties for accessing the `system` microservice. 
-
-Point your browser to the http://localhost:9085/query/systems/localhost URL. You can see the current OS name and Java version in JSON format. 
-
-Property expressions provide a way to embed expression segments in property values by using the `${...}` sequence. You can include the value of a configuration property in another configuration property by using the `${prop}` syntax. Additionally, you can implement expressions with the following syntax:
+The MicroProfile Config API is included in the MicroProfile dependency that is specified in your [hotspot file=0]`query/pom.xml` file. Look for the dependency with the [hotspot=mpconfig file=0]`microprofile` artifact ID. This dependency provides a library that allows you to use the MicroProfile Config API to externalize configurations for your microservices. The [hotspot=mpconfig file=1]`mpConfig` feature is also enabled in the [hotspot file=1]`query/src/main/liberty/config/server.xml` file.
 
-* `${prop:default}` - Provides a `default` value after the `:` character if the expression doesn't find a value for the `prop` property.
+In MicroProfile Config, property expressions provide a way to set and expand variables in property values. To use property expressions, define a property with a `${expression}` syntax, where `expression` is the name of another configuration property. When the property is loaded, the `${expression}` syntax is replaced with the value of the referenced property.
```

Comment:
i think this paragraph and the one at line 151 could be moved to the beginning of this section- that would set the context before the reader starts the config task

Reply:
Resolved in https://github.com/OpenLiberty/draft-guide-microprofile-config-apis/pull/17/commits/3d195954e4f0bf9f6a80e3696735774b6102941d